fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/AndOrValue.java

```diff
@@ -60,8 +60,20 @@
     protected final Operator operator;
 
     private enum Operator {
-        AND,
-        OR
+        AND("&&"),
```

Comment:
maybe just use `and` and `or` as SQL does that. either ways it's fine.

Reply:
Good point, thanks